src/popup/pages/MultisigProposalDetails.vue

```diff
@@ -0,0 +1,550 @@
+import MultisigProposalConsensus from '../components/MultisigProposalConsensus.vue';
+import AddressTruncated from '../components/AddressTruncated.vue';
+import Avatar from '../components/Avatar.vue';
+import TransactionTokens from '../components/TransactionTokenRows.vue';
+
+import AnimatedSpinner from '../../icons/animated-spinner.svg?skip-optimize';
+import ExternalLink from '../../icons/external-link.svg?vue-component';
+
+export default defineComponent({
+  components: {
+    PayloadDetails,
+    TransactionTokens,
+    MultisigProposalConsensus,
+    TransactionErrorStatus,
+    TransactionInfo,
+    AddressTruncated,
+    InlineTooltip,
+    Avatar,
+    TokenAmount,
+    DetailsItem,
+    LinkButton,
+    BtnMain,
+    CopyText,
+    AnimatedSpinner,
+    ExternalLink,
+  },
+  setup(props, { root }) {
+    const transaction = ref<IActiveMultisigTx>();
+    const revokingProposal = ref<boolean>(false);
+    const accounts = computed<IAccount[]>(() => root.$store.getters.accounts);
+    const getExplorerPath = computed(() => root.$store.getters.getExplorerPath);
+    const isTxAex9 = computed(() => root.$store.getters.isTxAex9);
+    const getTxSymbol = computed(() => root.$store.getters.getTxSymbol);
+    const getTxAmountTotal = computed(() => root.$store.getters.getTxAmountTotal);
+    const canCompleteTransaction = computed(
+      (): boolean => (
+        !!transaction.value
+        && transaction.value.confirmedBy.length >= transaction.value.confirmationsRequired
+      ),
+    );
+
+    const {
+      activeMultisigAccount,
+    } = useMultisigAccounts({ store: root.$store });
+
+    const {
+      fetchActiveMultisigTx,
+      revokeTx,
+      confirmTx,
+      sendTx,
+    } = useMultisigTransactions({
+      store: root.$store,
+    });
+
+    const {
+      setTransaction,
+      isErrorTransaction,
+      isAllowance,
+      tokens,
+    } = useTransaction({
+      store: root.$store,
+      showDetailedAllowanceInfo: true,
+    });
+
+    const isConfirmedByCurrentAccounts = computed(
+      () => !!accounts.value.filter(
+        (account) => transaction.value?.confirmedBy?.includes(account.address),
+      ).length,
+    );
+
+    async function getTransactionDetails(currentAccount: any) {
+      const result = await fetchActiveMultisigTx(currentAccount.contractId);
+      if (!result) {
+        return;
+      }
+      const _transaction = {
+        ...currentAccount,
+        ...result,
+        tx: {
+          ...result.tx,
+          type: SCHEMA.TX_TYPE.gaMeta, // TODO: refactor to the proper type
+        },
+      } as any;
+
+      setTransaction(_transaction);
+      transaction.value = _transaction;
+    }
+
+    function isLocalAccount(address: string) {
+      return accounts.value.some((account) => account.address === address);
+    }
+
+    async function onRevokeProposal() {
+      revokingProposal.value = true;
+      const { contractId, txHash } = transaction.value as any;
```

Comment:
Every function that is using `txHash` won't work. `txHash` is an object here you need to use `hash`, since it was converted to `hex`, in general we need to do such changes with `txHash` as well.